demos/openshift-install/java-client-installer.sh

```diff
@@ -98,17 +125,15 @@ while [ "$1" != "" ]; do
   shift
 done
 
-if [ "$OPENSHIFT_URL" == "" ]; then
-  echo "Missing value in --ocp-url - exiting"
-  exit 1
-fi
+validate
 
-if [ "$PROJECT_NAME" == "" ]; then
-  echo "Missing value in --project-name - exiting"
-  exit 1
-fi
+OPENSHIFT_URL=$(prepare_input "$OPENSHIFT_URL" "OpenShift URL")
+DOCKER_URL=$(prepare_input "$DOCKER_URL" "Docker URL")
```

Comment:
I would explicitly say that the URL is of the OpenShift Docker registry, because "Docker URL" could be interpreted into a few different things

```suggestion
DOCKER_URL=$(prepare_input "$DOCKER_URL" "OpenShift Docker registry URL")
```

Reply:
done